frontend/src/pages/projects/components/BulkActionExpandableSection.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ExpandableSection, Flex, Label, List, ListItemProps } from '@patternfly/react-core';
+
+interface BulkActionExpandableSectionProps {
+  title: string;
+  children: React.ReactElement<ListItemProps>[];
+}
+
+export const BulkActionExpandableSection: React.FC<BulkActionExpandableSectionProps> = ({
+  title,
+  children,
+}) => {
+  const [isExpanded, setIsExpanded] = React.useState(true);
+
+  return (
+    <ExpandableSection
+      isExpanded={isExpanded}
+      onToggle={(_e, expanded) => setIsExpanded(expanded)}
+      toggleContent={
+        <Flex alignItems={{ default: 'alignItemsCenter' }}>
+          <span className="pf-v5-u-mr-sm">{title}</span>
+
+          <Label color="blue" isCompact>
+            {children.length}
+          </Label>
+        </Flex>
+      }
+      isIndented
+      className="pf-v5-u-mt-sm pf-v5-u-mb-sm"
+    >
+      <List className="pf-v5-u-pl-md">{children}</List>
```

Comment:
We should probably avoid classNames of pf css sizes -- we are not getting pixel perfect mocks from UX... there is default PF for everything until PF makes a mistake and we need to adjust -- do not use PF classes because it will prevent us from migrating release to release as they decide these things for all apps.

Reply:
I agree there's a chance problems could be caused so I've removed these, but in general, I don't think margins and padding pose much of a threat to causing issues migrating to newer versions of PF. For 1, they're easy to search and replace, 2, these typically always remain the same pixel values version to version, and 3, at least with v4 -> v5, PF provides migration tools (codemods) for things like this.